public/resources/css/module/_card.scss

```diff
@@ -36,18 +36,25 @@
       padding: 0px;
       margin: 0px 0px 0px ($unit * 4);
 
-      a {
-        font-size: 14px;
-        display: block;
-        line-height: 24px;
-        text-decoration: none;
-        padding: 0px 4px;
-
-        &:hover {
-          text-decoration: underline;
-          background: $mist;
+      &:not(.hover-card-container) {
+        a {
+          font-size: 14px;
+          display: block;
+          line-height: 24px;
+          text-decoration: none;
+          padding: 0px 4px;
+
+          &:hover {
+            text-decoration: underline;
+            background: $mist;
+          }
         }
       }
+
+      &.hover-card-container {
+        margin-left: 0;
+        margin-top: $unit * 2;
+      }
```

Comment:
I think you should just replace the entire containers on the install page with the cards instead of putting them inside the card module. You should then be able to remove these extras styles. Thanks.